Editor/Resources/EditorWindow/Pages/EC2Page.uxml

```diff
@@ -0,0 +1,92 @@
+﻿<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" noNamespaceSchemaLocation="../../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="./EC2Page.uss" />
+    <ui:VisualElement name="ManagedEC2Content" class="tab__content">
+        <ui:Label name="Title" class="content__header" text="Managed EC2" />
```

Comment:
This UI doesn't appear to have been updated to use the template we created with the landing page. Review the landing page uxml and update this file to use the same style: https://github.com/aws/amazon-gamelift-plugin-unity/blob/feature/anywhere/Editor/Resources/EditorWindow/Pages/LandingPage.uxml

Reply:
updated